pkg/events/core.go

```diff
@@ -11900,7 +11900,7 @@ var CoreEvents = map[ID]Definition{
 		},
 		sets: []string{},
 		fields: []trace.ArgMeta{
-			{Type: "char*", Name: "hidden_process"},
+			{Type: "char*", Name: "hidden_dirent"},
```

Comment:
```suggestion
			{Type: "char*", Name: "dirent_name"},
```

Although the event is used to catch hidden processes, the arg field is the name of the dirent with zeroed inode.

The event, in my opinion, should be called `ZeroedInodes`, since whoever indirectly calls filldir64 may pass a zeroed inode. @NDStrahilevitz WDYT?

Reply:
I'm not sure if research uses this event "raw" or passes it to a signature for further inspection. So their input is the relevant one. @OriGlassman @oshaked1

Reply:
It's used as data filter.

Reply:
As pointed out by @geyslan in this [reference](https://www.kernel.org/doc/html/latest/filesystems/ext4/directory.html), always assuming that inode=0 is explicitly used by an attacker to hide an inode may lead to a false positive... if unused dir uses `inode=0` as described:
```
The end of the entire directory is of course signified by reaching the end of the file. Unused directory entries are signified by inode = 0.
```

Maybe one possibility is changing the name as Geyslan suggested.

There is one helper from research which is using the field interpreting as `hidden process` whereas should be `hidden dirent` or `unused dirent` or something similar.